.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+# https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/ci-configurations/github-actions.md
+
+name: Release
+
+on:
+  workflow_dispatch:
+    branches: [master]
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: npm ci
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.npm_token }}
+        run: |
+          npm i -g semantic-release@19
+          semantic-release
```

Comment:
I tried running it locally it showed the following
it failed dry run for som reason do you know why ?
Do I have to create a tag first ?

```
Shmac:serverless-finch shereef.marzouk$ semantic-release
[1:30:27 PM] [semantic-release] › ℹ  Running semantic-release version 19.0.2
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/npm"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "verifyConditions" from "@semantic-release/github"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "analyzeCommits" from "@semantic-release/commit-analyzer"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "generateNotes" from "@semantic-release/release-notes-generator"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "prepare" from "@semantic-release/npm"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/npm"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "publish" from "@semantic-release/github"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/npm"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "addChannel" from "@semantic-release/github"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "success" from "@semantic-release/github"
[1:30:27 PM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
[1:30:27 PM] [semantic-release] › ⚠  This run was not triggered in a known CI environment, running in dry-run mode.
[1:30:29 PM] [semantic-release] › ✖  An error occurred while running semantic-release: Error: Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master
From https://github.com/fernando-mc/serverless-finch
 ! [rejected]        v3.0.0     -> v3.0.0  (would clobber existing tag)
    at makeError (/usr/local/lib/node_modules/semantic-release/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/usr/local/lib/node_modules/semantic-release/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async fetch (/usr/local/lib/node_modules/semantic-release/lib/git.js:120:5)
    at async /usr/local/lib/node_modules/semantic-release/lib/branches/index.js:21:5
    at async pEachSeries (/usr/local/lib/node_modules/semantic-release/node_modules/p-each-series/index.js:8:23)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/lib/branches/index.js:20:3)
    at async run (/usr/local/lib/node_modules/semantic-release/index.js:65:22)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/index.js:268:22)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/cli.js:55:5) {
  shortMessage: 'Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master',
  command: 'git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master',
  escapedCommand: 'git fetch --tags --update-head-ok "https://github.com/fernando-mc/serverless-finch.git" "+refs/heads/master:refs/heads/master"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'From https://github.com/fernando-mc/serverless-finch\n' +
    ' ! [rejected]        v3.0.0     -> v3.0.0  (would clobber existing tag)',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Error: Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master
From https://github.com/fernando-mc/serverless-finch
 ! [rejected]        v3.0.0     -> v3.0.0  (would clobber existing tag)
    at makeError (/usr/local/lib/node_modules/semantic-release/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/usr/local/lib/node_modules/semantic-release/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async fetch (/usr/local/lib/node_modules/semantic-release/lib/git.js:120:5)
    at async /usr/local/lib/node_modules/semantic-release/lib/branches/index.js:21:5
    at async pEachSeries (/usr/local/lib/node_modules/semantic-release/node_modules/p-each-series/index.js:8:23)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/lib/branches/index.js:20:3)
    at async run (/usr/local/lib/node_modules/semantic-release/index.js:65:22)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/index.js:268:22)
    at async module.exports (/usr/local/lib/node_modules/semantic-release/cli.js:55:5) {
  shortMessage: 'Command failed with exit code 1: git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master',
  command: 'git fetch --tags --update-head-ok https://github.com/fernando-mc/serverless-finch.git +refs/heads/master:refs/heads/master',
  escapedCommand: 'git fetch --tags --update-head-ok "https://github.com/fernando-mc/serverless-finch.git" "+refs/heads/master:refs/heads/master"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'From https://github.com/fernando-mc/serverless-finch\n' +
    ' ! [rejected]        v3.0.0     -> v3.0.0  (would clobber existing tag)',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
```

Reply:
I manually created v3.0.1 locally and ran it again it gave the same exact output

Reply:
I don't know for sure, but I think this is only happening if your local branch/tags are not in sync with the remote for some reason (i.e. a force push). I don't think this should happen when it runs on CI

Reply:
ok we will just have to test it after merging with the next release